src/AMOEBA/amoeba_kspace.cpp

```diff
@@ -68,25 +68,31 @@ void PairAmoeba::moduli()
   int maxfft = MAX(nfft1,nfft2);
   maxfft = MAX(maxfft,nfft3);
 
-  double *array = new double[bsorder];
-  double *bsarray = new double[maxfft];
-
+  //double *array = new double[bsorder];
```

Comment:
If this is no longer used and replaced with an alternative, the corresponding commented out code should be removed before merging.